backend/Piece.py

```diff
@@ -0,0 +1,87 @@
+from enum import Enum
+
+
+class Turn(Enum):
+    BLACK = 1
+    WHITE = 2
+
+    @classmethod
+    def swap_turn(self, curr):
+        if curr == Turn.BLACK:
+            return Turn.WHITE
+        else:
+            return Turn.BLACK
+
+class Piece:
+    def __init__(self, color):
+        self.color = color
+        self.position = (-1, -1)
+    
+    def get_turn(self):
+        return self.color
+    def get_position(self):
+        return self.position
+    
+    def set_turn(self, color):
```

Comment:
This should be `set_color`. Turn is a concept that the GameManager worries about and involves switching. Color for a given piece is static.

Reply:
See previous.